OpenRA.Mods.Common/OpenRA.Mods.Common.csproj

```diff
@@ -572,6 +572,32 @@
     <Compile Include="UtilityCommands\CheckCodeStyle.cs" />
     <Compile Include="UtilityCommands\ReplayMetadataCommand.cs" />
     <Compile Include="Widgets\Logic\ReplayUtils.cs" />
+    <Compile Include="Widgets\BackgroundWidget.cs" />
```

Comment:
Sadly MonoDevelop does not automatically sort those alphabetically. I did it manually in https://github.com/OpenRA/OpenRA/pull/7564.